src/whatsnew/2.1.rst

```diff
@@ -20,11 +20,162 @@
     :depth: 1
     :local:
 
+.. _release/2.1.x/upgrade:
+
+Upgrade Notes
+=============
+
+* The deprecated (and broken) OAuth 1.0 implementation has been removed.
+
+* If user code reads or manipulates replicator document states,
+  consider using the ``[replicator] update_docs = true`` compatibility
+  parameter. In that case replicator will continue updating documents
+  with transient replication states. However, that will incure a
+  performance cost. Consider instead switching using
+  ``_scheduler/docs`` HTTP endpoint.
+
+* The ``stale`` parameter for views and ``_find`` has been deprecated in favour
+  of two new parameters: ``stable`` and ``update``. The old ``stale=ok``
+  behaviour is equivalent to ``stable=true&update=false``, and the old
+  ``stale=update_after`` behaviour is equivalent to ``stable=true&update=lazy``.
+  The deprecated ``stale`` parameter will be removed in CouchDB 3.0.
+
+* The ``[couchdb] max_document_size`` configuration parameter was renamed
+  to ``[httpd] max_http_request_size`` to reflect its actual implementation.
+  It has been enhanced by a new ``[couchdb] single_max_doc_size`` setting.
+  Both are documented in the shipped ``default.ini`` file.
+
+.. _release/2.1.1:
+
+Version 2.1.1
+=============
+
+General
+-------
+
+* :ghissue:`617`: CouchDB now supports compilation and running under Erlang/OTP
+  20.x.
+
+* :ghissue:``: The ``couch_peruser`` functionality is now *really* fixed.
+  Really.
+
+* :ghissue:`827`: The cookie domain for AuthSession cookies, used in a
+  proxy authentication configuration, can now be customized via the ini file.
+
+* :ghissue:`858`: It is now possible to modify shard maps for system databases.
+
+* :ghissue:`732`: Due to an Erlang bug (ERL-343), invalid paths can be
```

Comment:
Would it be possible to make `ERL-343` a link to the Erlang bug?  Maybe just a nice-to-have for anyone seeking more info.